src/current/_includes/v25.1/misc/session-vars.md

```diff
@@ -42,6 +42,7 @@
 | <a id="intervalstyle"></a> `intervalstyle` | The input string format for [`INTERVAL`]({% link {{ page.version.version }}/interval.md %}) values. Accepted values include `postgres`, `iso_8601`, and `sql_standard`. | The value set by the `sql.defaults.intervalstyle` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) (`postgres`, by default). | Yes | Yes |
 | <a id="is-superuser"></a> `is_superuser` | If `on` or `true`, the current user is a member of the [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role). | User-dependent | No | Yes |
 | <a id="large-full-scan-rows"></a> `large_full_scan_rows` | Determines which tables are considered "large" such that `disallow_full_table_scans` rejects full table or index scans of "large" tables. The default value is `1000`. To reject all full table or index scans, set to `0`. | User-dependent | No | Yes |
+| <a id="legacy-varchar-typing"></a> `legacy_varchar_typing` | **New in v25.1:** If `on`, type checking and overload resolution for [`VARCHAR`]({% link {{ page.version.version }}/string.md %}#related-types) types ignore overloads that cause errors, allowing comparisons between `VARCHAR` and non-`STRING`-like placeholder values to execute successfully. If `off`, type checking of these comparisons is more strict and must be handled with explicit type casts. If `off`, type checking of these comparisons is more strict and must be handled with explicit type casts. | `off` | Yes | Yes |
```

Comment:
nit: looks like the last sentence is duplicated

Reply:
Fixed.